docs/resources/workspace_user.md

```diff
@@ -0,0 +1,82 @@
+---
+subcategory: "Workspace"
+---
+
+# huaweicloud_workspace_user
+
+Manages a Workspace user resource within HuaweiCloud.
+
+## Example Usage
+
+### Create a user that never expires
+
+```hcl
+variable "user_name" {}
+variable "email_address" {}
+
+resource "huaweicloud_workspace_user" "test" {
+  name  = var.user_name
+  email = var.email_address
+
+  account_expires            = "0"
+  password_never_expires     = false
+  enable_change_password     = true
+  next_login_change_password = true
+  disabled                   = false
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `region` - (Optional, String, ForceNew) The region in which to create the Workspace user resource.
+  If omitted, the provider-level region will be used. Changing this will create a new resource.
+
+* `name` - (Required, String, ForceNew) Specifies the user name.
+  + Pure numeric: the valid length is between `1` and `20`.
+  + Non-pure numeric: the name can contain `1` to `20` characters, only letters, digits, hyphens (-), underscore (_) and
+  dots (.) are allowed. The name must start with a letter.
+
+  Changing this will create a new resource.
+
+* `email` - (Required, String) Specifies the email address of user. The value can contain `1` to `64` characters.
+
+* `description` - (Optional, String) Specifies the description of user. The maximum length is `255` characters.
+
+* `account_expires` - (Optional, String) Specifies the user's valid period configuration.
+  + Never expires: **0**.
+  + Expires at a certain time: account expires must in Milli-RFC3339 format like `yyyy-MM-ddTHH:mm:ss.000Z`.
```

Comment:
suggest to update the timestamp without millisecond

Reply:
done.